src/collections/Pages/index.ts

```diff
@@ -88,6 +89,7 @@ export const Pages: CollectionConfig<'pages'> = {
                 FormBlock,
                 ProductsListBlock,
                 FAQ,
+                TabCards
```

Comment:
```suggestion
                TabCards,
```
The missing comma currently breaks the build.